metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -87,8 +87,8 @@ def step_init(obj, run_id, step_name, passed_in_split_indexes, task_id):
     "times to attach multiple tags.",
 )
 @click.option(
-    "--namespace",
-    "namespace",
+    "--kfp_namespace",
```

Comment:
Minor: this is really minor, but I believe the format for the user facing CLI will be `kfp-namespace` (`-` vs `_`). All the option click options are formatted this way.

Reply:
yeah good catch. will fix